pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[project]
+name = "ga4gh.vrs"
+authors = [
+    {name = "Larry Babb", email = "lbabb@broadinstitute.org"},
+    {name = "Reece Hart", email = "reecehart@gmail.com"},
+    {name = "Andreas Prlic", email = "andreas.prlic@gmail.com"},
+    {name = "Alex Wagner", email = "alex.wagner@nationwidechildrens.org"},
+]
+readme = "README.md"
+description = "GA4GH Variation Representation Specification (VRS) reference implementation"
+license = {file = "LICENSE"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+    "Topic :: Scientific/Engineering :: Medical Science Apps.",
+]
+keywords = [
+    "bioinformatics",
+    "ga4gh",
+    "genomics",
+    "hgvs",
+    "spdi",
+    "variation"
+]
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+    "bioutils",
+    "canonicaljson",
+    "coloredlogs",
+    "numpy",
+    "pyyaml",
+    "pydantic~=2.1",
+]
+
+[project.optional-dependencies]
+extras = [
+    "psycopg2-binary",
+    "biocommons.seqrepo>=0.5.1",
+    "bioutils>=0.5.2",
+    "hgvs>=1.4",
+    "requests",
+    "dill~=0.3.7",
+    "click",
+]
+dev = [
+    # tests
+    "pytest",
+    "pytest-cov",
+    "pytest-vcr",
+    "vcrpy",
+    "pyyaml",
+    # style
+    "pylint",
+    "yapf",
+    "smart_open",
+    # docs
+    "sphinx",
+    "sphinx_rtd_theme",
+    "restview",
+    # idk?  TODO figure it out
+    "ipython",
+    "jedi<0.18",
```

Comment:
truly not sure what these are doing here

Reply:
@jsstevenson I would delete these, I don't know what jedi is used for, I don't think it's necessary. `ipython` is only needed for notebooks